### PR TITLE
Lgrs issue

### PR DESCRIPTION
**Pull request by jonnyh404**

### CODE REVIEW

**Review by cflynn3 — APPROVED** [review]
Looks good to me.

File: ResSimpy/Nexus/structured_grid_operations.py [hunk 1]
```diff
@@ -355,7 +364,9 @@ def __extract_mod_positions(line_indx: int, file_as_list: list[str]) -> dict[str
                     mod_start_end['MOD'] = [[i + 1, len(file_as_list)]]
                 found_end_of_mod_table = False
                 for j in range(i + 1, len(file_as_list)):
-                    for keyword in STRUCTURED_GRID_KEYWORDS + GRID_ARRAY_KEYWORDS:
+                    keywords_to_stop_on = [x for x in (STRUCTURED_GRID_KEYWORDS + GRID_ARRAY_KEYWORDS)
+                                           if x != 'INCLUDE']
```
Comment (cflynn3) [inline ResSimpy/Nexus/structured_grid_operations.py:368]: Wouldn't it be simpler + more performant to do:
`keywords_to_stop_on = STRUCTURED_GRID_KEYWORDS + GRID_ARRAY_KEYWORDS
keywords_to_stop_on.remove("'INCLUDE'")
`

**Review by cflynn3 — APPROVED** [review]
Latest changes look good.

**Review by obi-isebor — APPROVED** [review]
LGTM

**Pull request merged**